app/messaging/back-office/process-backoffice.js

```diff
@@ -1,26 +1,25 @@
 const { getAll, getApplicationCount } = require('../../repositories/application-repository')
 const { backOfficeResponseMsgType, backOfficeResponseQueue } = require('../../config')
 const sendMessage = require('../../messaging/send-message')
-const states = require('../states')
 
 const processBackOfficeRequest = async (msg) => {
   const responseMessage = msg.body
   const { sessionId } = msg
   try {
-    // Get All Applications
-    const result = await getAll(msg.body.search.text, msg.body.limit, msg.body.offset)
+    // Search application with SBI number and return requested page for selected offset and limit.
+    const result = await getAll(msg.body.search.text, msg.body.offset, msg.body.limit)
     const total = await getApplicationCount(msg.body.search.text)
     if (result.length <= 0) {
```

Comment:
Can this `if` block be removed?
If `result` is an array of the applications and `total` is a number then I think you can just have
```js
await sendMessage({ applications: result, total }, backOfficeResponseMsgType, backOfficeResponseQueue, { sessionId })
```

Or, `result` is renamed to `applications` even simpler:
```js
await sendMessage({ applications, total }, backOfficeResponseMsgType, backOfficeResponseQueue, { sessionId })
```